WBooks/Book Details/BookDetailsView.swift

```diff
@@ -42,6 +42,7 @@ class BookDetailsView: UIView, NibLoadable {
         didSet {
             self.rent.layer.cornerRadius = 20
```

Comment:
Unnecessary `self`. Remove it from all properties you use it.

Reply:
Done!